src/league.js

```diff
@@ -576,7 +606,7 @@ league_attributes = {
     //--------------------------------------------------------------------------
     'formatModsResponse': function() {
         var self = this;
-        moderators = _.map(self.options.moderators, function(name) {
+        moderators = _.map(self._moderators, function(name) {
             return name[0] + "\u200B" + name.slice(1);
         });
         return Q.fcall(function() {
```

Comment:
I was thinking that this one should point out that it does _not_ ping the moderators, and should suggest the command that does ping them.  

```Lonewolf mods: endrawes0, jptriton, cyanfish, lakinwecker (**Note**: this does not ping the moderators. Use: `@chesster: summon mods` to ping them```